src/wms/src/main/java/org/geoserver/wms/capabilities/Capabilities_1_3_0_Transformer.java

```diff
@@ -1,4 +1,4 @@
-/* (c) 2014 - 2015 Open Source Geospatial Foundation - all rights reserved
+/* (c) 2014 - 2016 Open Source Geospatial Foundation - all rights reserved
```

Comment:
I am not sure if the headers need to be updated anymore, I think the proposal passed but the developers guide is not fixed up yet :)  @bencaradocdavies  can you clarify?